test/Microsoft.SqlTools.ServiceLayer.UnitTests/ObjectExplorer/ServerVersionHelperTests.cs

```diff
@@ -168,7 +168,8 @@ private void VerifyCalculateServerTypeForEngineEdition(int engineEdition, SqlSer
             };
 
             SqlServerType actual = ServerVersionHelper.CalculateServerType(serverInfo);
-            Assert.Equal(expected, actual);
+            
+            Assert.True(actual.ToString().Equals(expected.ToString()), "Verify correct engine edition number");
```

Comment:
maybe print expected and actual values in the message

Reply:
Yes, please do. The message is mostly useless if you don't also have the expected/actual values since you'd have to debug in to find that out then regardless.